### ISSUE TITLE
[North Star] With screen reader, when number question is in an error state, it is impossible to type an answer

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#10210]
**Describe the bug**
With VoiceOver on, when a number question is in an error state, typing numbers causes the screen reader to make a beeping sound and it says "Header not found".

**To Reproduce**

1. Add a required number question to a program and publish.
2. Go to the applicant home page and start applying to the program.
3. Don't fill out an answer for the number question and hit "Continue".
4. Turn the screen reader on.
5. Navigate to the number question input box.
6. Try entering a number.
7.  You will hear a beep and "Header not found".

**Expected behavior**
The user should be able to enter a number into the question input field.

**Screenshots**

https://github.com/user-attachments/assets/79279875-a070-49e1-af19-302728480317

**Desktop (please complete the following information):**

- OS: Mac, desktop
- Browser: Chrome
- Screen reader: VoiceOver

**Comment by watermandrewc** [comment]
I was able to reproduce this on non-number questions too (e.g. a Name question); I think it's related to Quick Nav if you're on Safari on a Mac. For instance, if I turn single-key Quick Nav off when this happens, it resolves and I'm able to type again. Steps:
1. Go to a required question and click "Continue" without filling the question out.
2. Navigate to (one of) the boxes with errors, then turn on screen reader.
3. You won't be able to type due to bug noted above (e.g. "Header not found" gets read aloud instead).
4. Turn QuickNav off (likely with Ctrl + Opt + Q: https://support.apple.com/guide/voiceover/with-quick-nav-vo27943/mac).
5. Once single-key QuickNav is off, I'm able to type normally.

Steps 1-3 can be tricky to reproduce; I find that to repro, I have to specifically turn voiceover on right after I've clicked "Continue" and focus is automatically applied to the first input field with an error. It doesn't repro 100% of the time.

**Comment by melanie-exygy** [comment]
> I was able to reproduce this on non-number questions too (e.g. a Name question); I think it's related to Quick Nav if you're on Safari on a Mac. For instance, if I turn single-key Quick Nav off when this happens, it resolves and I'm able to type again. Steps:
> 
> 1. Go to a required question and click "Continue" without filling the question out.
> 2. Navigate to (one of) the boxes with errors, then turn on screen reader.
> 3. You won't be able to type due to bug noted above (e.g. "Header not found" gets read aloud instead).
> 4. Turn QuickNav off (likely with Ctrl + Opt + Q: https://support.apple.com/guide/voiceover/with-quick-nav-vo27943/mac).
> 5. Once single-key QuickNav is off, I'm able to type normally.
> 
> Steps 1-3 can be tricky to reproduce; I find that to repro, I have to specifically turn voiceover on right after I've clicked "Continue" and focus is automatically applied to the first input field with an error. It doesn't repro 100% of the time.

Thanks @watermandrewc !  I'm still only able to reproduce with the number question, but turning QuickNav off fixed the issue.  Since the issue is related to a VoiceOver setting rather than our code, I'm going to close the bug.  Really appreciate you investigating this!
